server/controllers/postMessage.js

```diff
@@ -0,0 +1,50 @@
+import express from 'express';
+import PostIt from './../src/PostIt';
+import verifyToken from './../middleware/verifyToken';
+import verifyMembership from './../middleware/verifyMembership';
+
+const postMessage = express.Router();
+const postIt = new PostIt();
+
+/**
+ * Posts  a message to a group
+ */
+postMessage.post(
+  '/groups/:groupId/messages',
+  verifyToken,
+  verifyMembership,
+  (request, response) => {
+    const { priority, body } = request.body;
+    if (!request.body.body) {
+      response.status(400).json({
+        message: 'Message body is required'
+      });
+    } else {
+      postIt.postMessage(
```

Comment:
Try and handle more errors here. What if there's no group with the supplied groupId?

Reply:
I handled enough errors. Pls check `verifyMembership.js`